empress_gui.py

```diff
@@ -10,8 +10,8 @@
 import pathlib
 
 import matplotlib
-# use tkinter backend, see # https://matplotlib.org/faq/usage_faq.html#what-is-a-backend
-matplotlib.use("tkagg")
+# use tkinter backend, required for embedding in Tk canvas
```

Comment:
Please remove these comments too. We are not using tkinter backend for matplotlib anymore.